src/buildCommon.js

```diff
@@ -72,7 +72,7 @@ const makeSymbol = function(
     let symbolNode;
     if (metrics) {
         let italic = metrics.italic;
-        if (mode === "text") {
+        if (mode === "text" || (options && options.font === "mathit")) {
             italic = 0;
```

Comment:
Interesting. The combining of text glyphs we do should have a similar affect, shouldn't it. I guess it still keeps the italic value from the last glyph so it still changing the spacing at the boundaries between font types whereas this won't.

Reply:
Not sure, but it's something we could investigate.